### PR TITLE
Test fix poshi test

### PR DESCRIPTION
**Pull request by gustavolimav**

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:help

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
FALHOU TUDOOOOOOOOOOOOOOOOOO

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:report:2561232177

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:search

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SearchContinuousUpgrade.testcase [hunk 1]
```diff
@@ -388,11 +392,15 @@ definition {
...
 		Blueprints.addElement(element = "Limit Search to These Sites");
 
-		Type(
-			key_elementName = "Limit Search to These Sites",
-			key_fieldLabel = "Group IDs",
-			locator1 = "Blueprints#BLUEPRINTS_ADDED_ELEMENT_INPUT_FIELD",
-			value1 = 12345);
+		Click(locator1 = "Button#SELECT");
+
+		var siteName = TestCase.getSiteName(siteName = ${siteName});
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SearchContinuousUpgrade.testcase:397]: Declare this variable before it is used (before line 401)

**Pull request closed without merging**